cdap-api/src/main/java/co/cask/cdap/api/common/RuntimeArguments.java

```diff
@@ -74,33 +76,49 @@ private RuntimeArguments() {
   }
 
   /**
-   * Extract all arguments for a given scope.
+   * Extracts the scope from the argument as identified by its prefix.
```

Comment:
Better say: Identifies arguments with a given scope prefix and adds them back without the scope prefix.